spec/helpers/WHY-THIS-JAR.txt

```diff
@@ -0,0 +1,4 @@
+The derbyrun.jar is used by spec/filters/jdbc_static_file_local_spec.rb via
```

Comment:
Why do you need a separate Derby server - as you are using embedded Derby, it will automatically create the Derby instance in the VM running the tests

Reply:
That test jdbc_static_file_local_spec turns the local vs remote around and tests the operation of the local db being an on-machine db server and the remote being in JVM. This will be a supported feature soonish.